### ISSUE TITLE
Issue reporter window can be easily sent to background

### ISSUE THREAD

**Issue by rhauch** [confluentinc/vscode#222]
Type: <b>Bug</b>

Steps to reproduce:
1. Click on `Report an Issue` in the `Support` panel of Confluent for VS Code. This opens the `Issue Reporter` window.
2. Click the main VS Code window (or any other application window that might hide the `Issue Reporter` window)
3. The `Issue Reporter` window is hidden. 
4. Clicking `Report an Issue` does not bring the `Issue Reporter` window in focus.

To resolve, you can choose any of these:
1. Go to `Window` menu, and select `Issue Reporter`. 
2. Use OS-specific tools (e.g., Mission Control on macOS) to show hidden windows, and show `Issue Reporter`.
3. In task bar, chose windows for VS Code and pick `Issue Reporter`.

Desired behavior:
- Clicking `Report an Issue` again with `Issue Reporter` behind VS Code workspace window brings the existing `Issue Reporter` window to the front.

There is an [older issue in VS Code itself](https://github.com/microsoft/vscode/issues/130497) describing this, but the issue was supposedly fixed by [this VS Code PR](https://github.com/microsoft/vscode/pull/156141) back in Aug 2022 by listing the `issue Reporter` in the taskbar (on Windows and macOS).

It's fine to close this if this is a core behavior of VS Code.

Extension version: 0.15.3
VS Code version: Code 1.93.1 (Universal) (38c31bc77e0dd6ae88a4e9cc93428cc27a56ba40, 2024-09-11T17:20:05.685Z)
OS version: Darwin x64 22.6.0
Modes:

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i9-9880H CPU @ 2.30GHz (16 x 2300)|
|GPU Status|2d_canvas: enabled<br>canvas_oop_rasterization: disabled_off<br>direct_rendering_display_compositor: disabled_off_ok<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>opengl: enabled_on<br>rasterization: enabled<br>raw_draw: disabled_off_ok<br>skia_graphite: disabled_off<br>video_decode: enabled<br>video_encode: enabled<br>webgl: enabled<br>webgl2: enabled<br>webgpu: enabled<br>webnn: disabled_off|
|Load (avg)|3, 2, 2|
|Memory (System)|32.00GB (0.40GB free)|
|Process Argv|--crash-reporter-id 2e48c337-d272-4940-930b-5364e4eb09ff|
|Screen Reader|no|
|VM|0%|
</details><details>
<summary>A/B Experiments</summary>

```
vsliv368cf:30146710
vspor879:30202332
vspor708:30202333
vspor363:30204092
vscod805:30301674
binariesv615:30325510
vsaa593cf:30376535
py29gd2263:31024239
c4g48928:30535728
azure-dev_surveyone:30548225
2i9eh265:30646982
962ge761:30959799
pythongtdpath:30769146
welcomedialogc:30910334
pythonnoceb:30805159
asynctok:30898717
pythonmypyd1:30879173
h48ei257:31000450
pythontbext0:30879054
accentitlementsc:30995553
dsvsc016:30899300
dsvsc017:30899301
dsvsc018:30899302
cppperfnew:31000557
dsvsc020:30976470
pythonait:31006305
dsvsc021:30996838
g316j359:31013175
a69g1124:31058053
dvdeprecation:31068756
dwnewjupyter:31046869
impr_priority:31102340
nativerepl1:31139838
refactort:31108082
pythonrstrctxt:31112756
flightc:31134773
wkspc-onlycs-t:31132770
nativeloc2:31134642
wkspc-ranged-t:31125599
cf971741:31134768
ei213698:31121563
iacca2:31138163

```

</details>

**Comment by shouples** [comment]
@rhauch I don't know how much control we have over this one, since it's built-in VS Code functionality and we're just sending it the extension name to auto-fill the `Extension` dropdown: https://github.com/confluentinc/vscode/blob/3f6ef1e4005a273abd5184178bbfa650b842304c/src/commands/support.ts#L18

**Comment by rhauch** [comment]
Thanks, @shouples. Do you think it's worth trying to open an issue so that commands like:
```
vscode.commands.executeCommand("vscode.openIssueReporter", "confluentinc.vscode-confluent"); 
```
would bring any existing `Issue Reporter` window to the front? I couldn't find an existing issue like that [with this filter](https://github.com/microsoft/vscode/issues?q=is%3Aissue+%22vscode.openIssueReporter%22+), but there are a ton of open issues; e.g., 40K+ open issues from [this filter with just `Issue Reporter`](https://github.com/microsoft/vscode/issues?q=is%3Aissue+%22Issue+Reporter%22).

**Comment by shouples** [comment]
@rhauch I think it's worth bringing up at least; there are some other "Focus" type commands that are built-in, and the functionality seems to exist for editor windows to be brought back into focus.

Filed https://github.com/microsoft/vscode/issues/229107

**Comment by shouples** [comment]
@rhauch Quick follow-up: I think this may actually be a bug in the new experimental setting released in 1.93: https://code.visualstudio.com/updates/v1_93#_new-issue-reporter-implementation

As soon as I disabled the `[issueReporter.experimental.auxWindow](vscode://settings/issueReporter.experimental.auxWindow)` setting, clicking our "Report an Issue" item brought the window back into focus.

Filed a [bug issue](https://github.com/microsoft/vscode/issues/229152) and closed the previous feature request issue.

**Comment by shouples** [comment]
Going to close this one since it's out of our control.

For anyone else that sees this same behavior (starting from VS Code 1.93.0), I recommend going into the extension settings `Cmd+,`/`Ctrl+,` and searching for `@id:issueReporter.experimental.auxWindow` and disabling it.
<img width="711" alt="image" src="https://github.com/user-attachments/assets/04a7f0d1-1b1d-4310-8447-d3122431b2aa">
